src/middlewares/logger.middleware.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable, Logger, NestMiddleware } from '@nestjs/common';
+import { Request, Response, NextFunction } from 'express';
+import { JwtService } from '@nestjs/jwt';
+
+@Injectable()
+export class LoggerMiddleware implements NestMiddleware {
+  private readonly logger = new Logger();
+  private readonly jwt = new JwtService();
```

Comment:
PullProofAI

Consider injecting the `JwtService` into the `LoggerMiddleware` constructor instead of creating a new instance within the class.
```typescript
6: export class LoggerMiddleware implements NestMiddleware {
7:   private readonly logger = new Logger();
8:   constructor(private readonly jwt: JwtService) {}
```
By injecting the service, you can improve testability and make it easier to manage dependencies.


<!-- This is an auto-generated comment by PullProofAI -->